django/core/jinja2/base.jinja

```diff
@@ -272,6 +272,23 @@
     {% endwith %}
 {% endmacro %}
 
+{% macro ogp_tags_base(url=None, title=None, description=None, image=None) %}
+    <!-- Facebook Meta Tags -->
+    <meta property="og:url" content="{{ url or 'https://www.comses.net/'}}">
+    <meta property="og:type" content="website">
+    <meta property="og:title" content="{{ title or 'The Network for Computational Modeling in the Social and Ecological Sciences (CoMSES Net)'}}">
+    <meta property="og:description" content="{{ description or 'CoMSES Net is an international open research community dedicated to fostering good practices for computational / agent based modeling.'}}">
+    <meta property="og:image" content="{{ image or static('images/logo-comses.png') }}">
+
+    <!-- Twitter Meta Tags -->
+    <meta name="twitter:card" content="summary_large_image">
+    <meta property="twitter:domain" content="comses.net">
+    <meta property="twitter:url" content="{{ url or 'https://www.comses.net/'}}">
+    <meta name="twitter:title" content="{{ title or 'The Network for Computational Modeling in the Social and Ecological Sciences (CoMSES Net)'}}">
+    <meta name="twitter:description" content="{{ description or 'CoMSES Net is an international open research community dedicated to fostering good practices for computational / agent based modeling.'}}">
+    <meta property="og:image" content="{{ image or static('images/logo-comses.png') }}">
+{% endmacro %}
```

Comment:
I think this should go in core/common.jinja with most of the other shared macros and then be explicitly imported where we use it